### ISSUE TITLE
Don't develop prioritizing the GUI: Always update all script Examples!

### ISSUE THREAD

**Issue by lvanfretti** [SmarTS-Lab/iTesla_RaPId#24]
For some reason, my old examples are not available, and more importantly, non of the new examples have any scripts to execute RaPId without a GUI.

I hate GUIs!!! We designed the software from the beginning to be independent from it.
Now the software has evolved, and I can't run my examples... it seems like @janlav and @GiuseppeLaera have done most of the development focusing on the use of the GUI. This is really great, but we should focus on NOT only developing the functionalities and updating the GUI, but most importantly, preserving and updating the script examples. 

It is very important we have all examples with scripts.

I have asked @tinrabuzin that my old SecondOrder example should be automatically launched and tested at installation, so this would allow to check through scripts that everything is working fine.

We also need to develop a testing script that runs through all of the examples in the Examples folder and makes sure they are still compatible with the changes made to the source.

To summarize:
  - @tinrabuzin use scripting example to be executed together with installation
  - @janlav document all the information contained in RapidObject and what is needed to run a script. Also, build the automatic testing code to run through all the examples to check if they are working. Set the examples to be executed for a couple of iterations...

@janlav we latter need to discuss how to build a test code to check that all the optimizes are working...

**Comment by MaximeBaudette** [comment]
@lvanfretti 
You raise several points in this "new issue". And while they are all related, each and every one are independent tasks, that make it hard to follow up in the context of GitHub issues. 

Also several of the points that you mention are duplicated from older github issues: 
- #20 on updating the second order model 
- #12 on running an example in the installation script
- #13 on documentation issues
- #16 on rapid_CMD usage

**Comment by lvanfretti** [comment]
@MaximeBaudette as usual you are way better than me modularizing what needs to be done! Thank you!!!

**Comment by MaximeBaudette** [comment]
Ok now I have dispatched all your comments into the existing issues, I'm closing this one.
